packages/platform/libraries/index.ts

```diff
@@ -119,3 +124,8 @@ export type { TFindTeamMembersMatchingAttributeLogicInputSchema } from "@calcom/
 export { checkAdminOrOwner } from "@calcom/features/auth/lib/checkAdminOrOwner";
 
 export { verifyPhoneNumber, sendVerificationCode };
+export { getLocationValuesForDb };
+export { getDefaultConferencingAppLocation };
```

Comment:
Would be good to move this to `packages/platform/libraries/conferencing.ts`  and check if you can move other newly exported functions also to specific files, or you can createa new one like `locations.ts` to export `getLocationValueForDB`, `getBookingDataLocation` etc - we should try not to export from `index.ts` files of platform libraries if we can.